Consul/Consul.csproj

```diff
@@ -1,7 +1,7 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461;netstandard2.1</TargetFrameworks>
```

Comment:
It seems that `netstandard2.1` is not implementing enough of `dynamic` type support to completely remove the "Microsoft.CSharp" dependency, I think it would work with `net6.0` instead of `netstandard2.1`. 
I wonder though, if it is worth adding a new target framework, just to kill one NuGet dependency?

Reply:
have added net 6 instead and at the same time improved the cancellation toke propagation.. Hence closing conversation.